.circleci/config.yml

```diff
@@ -76,3 +96,9 @@ workflows:
           filters:
             branches:
               only: master
+      - deploy-alpha:
```

Comment:
This uses the `deploy-alpha` job for now, but we'll update this to `deploy-beta` when we release the beta